test/harvester/seed.js

```diff
@@ -0,0 +1,111 @@
+var _ = require('lodash');
+var inflect = require('i')();
+var request = require('supertest');
+var RSVP = require('rsvp');
+var Promise = RSVP.Promise;
+
+var config = require('../config.js');
+var fixtures = require('./fixtures');
+
+/**
+ * Configure seeding service.
+ *
+ * @param configuration optional seeding configuration; currently baseUrl of harvester app is required;
+ * @returns {{beforeEach: Function, before: Function}} configured seeding service
+ */
+module.exports = function (configuration) {
```

Comment:
We don't need configuration because we can read harvester URL from app instance.